src/Adapter/Module/AdminModuleDataProvider.php

```diff
@@ -248,9 +248,9 @@ public function isAllowedAccess($action, $name = '')
 
     /**
      * @param AddonsCollection $addons
-     * @param null $specific_action
+     * @param null|string $specific_action
```

Comment:
`null` comes in last position

```suggestion
     * @param string|null $specific_action
```